src/components/Network.js

```diff
@@ -129,10 +145,12 @@ class Network extends React.Component {
             </div>
             <ul className="list-group list-group-flush">
               <li className="list-group-item">
-                <div>
-                  Synced timestamp: {dateFormatter.timestampToString(sync.synced_timestamp)}<br />
-                  Latest timestamp: {dateFormatter.timestampToString(sync.latest_timestamp)}
-                </div>
+                {sync_v1 ? (
+                  <div>
+                    Synced timestamp: {dateFormatter.timestampToString(sync_v1.synced_timestamp)}<br />
```

Comment:
I'd like to have at least the latest timestamp in sync-v2. Is it possible? Maybe we need to change the design.

Reply:
It's possible when we have the block of the remote peer, which is almost always the case, but not always, but it is technically possible to still have it.

But I'd rather make this something specific to sync-v2 rather than making it generic into the `sync` information, which doesn't have a design yet. I also don't quite understand the practical advantage of showing the timestamp instead of the height+hash (hich is what the sync-v2 will be able to show), timestamp is indirectly accessible.

In the odd case where a node might be mining in the future we wouldn't know the timestamp either, which is the only case I can think of where actually having the timestamp would be useful.

Reply:
Now I conditionally display `synced timestamp` and `latest timestamp`, so unless you want to not display timestamps, this requested change is resolved.